include/linker/linker-defs.h

```diff
@@ -163,6 +163,10 @@ extern char _image_rodata_size[];
 extern char _vector_start[];
 extern char _vector_end[];
 
+#ifdef CONFIG_SW_VECTOR_RELAY
```

Comment:
Is this now needed @e-rk ?

Reply:
This address will be used in mcuboot main.c

Reply:
ok thanks